src/components/books/BookList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Book from './Book';
+
+const BookList = () => {
+  const bookstore = [{
+    genre: 'Action',
+    title: 'Snatch',
+    author: 'Guy Pearce',
+    id: '1',
+  }];
+
+  return (
+    <ul className="book-list">
+      {bookstore.map((book) => (
```

Comment:
- [ ] It looks like you're listing the books but I'm not able to see it when I start the application, please, show the books to check how this works on the application. 🐶 
![image](https://user-images.githubusercontent.com/20563252/130534003-33a34882-386a-4fa9-9818-992b40dc208d.png)